content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -11,10 +11,33 @@ The rules engine can radically boost your productivity in the PIM by automating
 
 The rules are defined in a YML file that you can import in the PIM, using a dedicated job profile.   
 
-A rule is defined by a code (required) and you can name it with a label as well. This label can have a value per locale.
-
 This feature is very, like, really powerful, and the use cases are countless. In order to guide you in the rules engine usage, we wrote this article where you will find out what you can use it for and how it works.
 
+# Name your rules
+
+A rule is defined by a code (required) and you can name it with a label as well. This label can have a value per locale.
```

Comment:
```suggestion
Each rule must have a unique code. It can also have a label per enabled PIM locale. This label will appear under `Settings`/`Rules` and be also visible in your product form under your smart attributes.
```